libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -238,4 +248,85 @@ export class DhActorsCreateActorModalComponent {
   close() {
     this.innerModal?.close(false);
   }
+
+  createMarketParticipent(): void {
+    this._apollo
+      .mutate({
+        mutation: CreateMarketParticipantDocument,
+        variables: {
+          input: {
+            userInvite: {
+              email: this.newUserForm.controls.email.value,
+              firstName: this.newUserForm.controls.firstName.value,
+              lastName: this.newUserForm.controls.lastName.value,
+              phoneNumber: this.newUserForm.controls.phone.value,
+              assignedRoles: this._userSelectedUserRoles.map((userRole) => userRole.id),
+              assignedActor: 'f3df856f-bd11-4174-97cb-fb6bc54c300a',
+            },
+            organizationId: this.chooseOrganizationForm.controls.orgId.value,
+            organization: this.chooseOrganizationForm.controls.orgId.value
+              ? null
+              : {
+                  address: {
+                    country: this.newOrganizationForm.controls.country.value,
+                  },
+                  domain: this.newOrganizationForm.controls.domain.value,
+                  businessRegisterIdentifier: this.newOrganizationForm.controls.cvrNumber.value,
+                  name: this.newOrganizationForm.controls.companyName.value,
+                },
+            actorContact: {
+              email: this.newActorForm.controls.contact.controls.email.value,
+              name: this.newActorForm.controls.contact.controls.departmentOrName.value,
+              phone: this.newActorForm.controls.contact.controls.phone.value,
+              category: ContactCategoryType.Default,
+            },
+            actor: {
+              name: { value: this.newActorForm.controls.name.value },
+              organizationId:
+                this.chooseOrganizationForm.controls.orgId.value ??
+                'c3df856f-bd11-4174-97cb-fb6bc54c300b',
```

Comment:
issue: hard-coded value.

Reply:
They are needed but, cannot be provided. API needs to be changed at a later date